builder/vmware/common/driver_esx5.go

```diff
@@ -679,17 +679,28 @@ func (d *ESX5Driver) Download(src, dst string) error {
 	return d.comm.Download(d.datastorePath(src), file)
 }
 
-func (d *ESX5Driver) VerifyChecksum(ctype string, hash string, file string) bool {
-	if ctype == "none" {
+func (d *ESX5Driver) VerifyChecksum(hash string, file string) bool {
+	if hash == "none" {
 		if err := d.sh("stat", strconv.Quote(file)); err != nil {
 			return false
 		}
-	} else {
-		stdin := bytes.NewBufferString(fmt.Sprintf("%s  %s", hash, file))
-		_, err := d.run(stdin, fmt.Sprintf("%ssum", ctype), "-c")
-		if err != nil {
-			return false
-		}
+		return true
+	}
+
+	// TODO(azr): this will not work with checksums from files
+	vs := strings.SplitN(hash, ":", 2)
+	var ctype string
+	switch len(vs) {
+	case 2:
+		ctype, hash = vs[0], vs[1]
+	default:
+		return false
```

Comment:
So if the user leaves the ctype off the input checksum, we'll _always_ upload?

Reply:
Ah good catch, here I plan on trashing that whole func and simply using the getter.Checksum Moss created, this way we wont depend on the existence of a `fmt.Sprintf("%ssum", ctype)` bin